centml/cli/login.py

```diff
@@ -95,6 +96,16 @@ def login(token_file):
     if token_file:
         auth.store_centml_cred(token_file)
 
+    # Try client credentials flow first if SERVICE_ACCOUNT_ID and SERVICE_ACCOUNT_SECRET are set
+    if settings.CENTML_SERVICE_ACCOUNT_ID and settings.CENTML_SERVICE_ACCOUNT_SECRET:
+        click.echo("Authenticating with client credentials...")
+        access_token = auth.authenticate_with_client_credentials()
+        if access_token:
+            with get_centml_client() as cclient:
+                cclient.initialize_user()
+            click.echo("✅ Login successful with client credentials")
+            return
+
```

Comment:
lets implement the entire logic in sdk/auth.py

Reply:
we try not to change any other part of the code

Reply:
Moved most of the logic to sdk/auth.py. However, we still need a way to initialize a client, so I get that one function call here